source/antwerp/tier2_hardware.rst

```diff
@@ -1,12 +1,270 @@
+   you can also use ``module spider`` using a specific module version. 
+
+   The explanation given about the module(s) that need to be loaded, 
+   can be a bit confusing. You have to take only one line from the output and load
+   all modules on that line.
+
+   |Example| Using ``module spider`` with a version 
+     If you want more specific information on ``Python/3.11.3-GCCcore-12.3.0``, 
+     you can use 
+    
+     .. code-block:: bash
+ 
+        $ module spider Python 3.11.3-GCCcore-12.3.0
+          -----------------------------------------------------------------
+          Python: Python/3.11.3-GCCcore-12.3.0
+          -----------------------------------------------------------------
+            Description:
+              Python is a programming language that lets you work more
+              quickly and integrate your systems more effectively.
+       
+       
+            You will need to load all module(s) on any one of the lines below 
+            before the "Python/3.11.3-GCCcore-12.3.0" module is available to load.
+        
+              calcua/2023a
+              calcua/all
+ 
+     In this case, it suffices to first load either ``calcua/2023a``, 
+     or ``calcua/all``: it is not required to load both.
+
+* Search through the whatis-information
+   Each module also contains a limited amount of additional information that can be shown with
+   the ``module whatis`` command, e.g.
+
+   .. code-block:: bash
+
+      module whatis Python/3.11.3-GCCcore-12.3.0
+
+   You can search through this 'whatis'-information of installed modules using ``module keyword``
+
+   .. code-block:: bash
+
+      module keyword CMake
+
+If you did not find the software you want to use, please do not hesitate to contact 
+the UAntwerp support team at hpc@uantwerpen.be.
+
+Installation directories
+========================
+  
+Additional packages or software distributions should be installed 
+on VSC_DATA or VSC_SCRATCH, depending on the circumstances.
+
+* |UA| **Users with a vsc2xxxx account**
+   
+   When working on the UAntwerp clusters, VSC_DATA is a local file
+   system for you. We expect that for :ref:`Python <Python packages>` packages 
+   (and similarly for :ref:`R <R_package_management>` and :ref:`Perl <Perl packages>`), 
+   **VSC_DATA** will give better performance than VSC_SCRATCH due to the amount of file 
+   metadata accesses that occur when running Python.
+
+   |Example| To install the ``sphinx`` Python package, you can use the following command
+   
+   .. code-block:: bash
+
+      pip install --prefix=${VSC_DATA}/python_lib sphinx
+* |VUB| |UG| |KUL| **Users with a different home institution**
+        
+   VSC_DATA is a file system at your local
+   institution. Due to the distance between our clusters and your home institution, file access to 
+   VSC_DATA will have a high latency. 
+   
+   Installing the packages that you need locally on **VSC_SCRATCH**
+   will give much better performance.
```

Comment:
There is a formatting issue here with the indentation:
```suggestion
* |UA| **Users with a vsc2xxxx account**
   
  When working on the UAntwerp clusters, VSC_DATA is a local file
  system for you. We expect that for :ref:`Python <Python packages>` packages 
  (and similarly for :ref:`R <R_package_management>` and :ref:`Perl <Perl packages>`), 
  **VSC_DATA** will give better performance than VSC_SCRATCH due to the amount of file 
  metadata accesses that occur when running Python.

  |Example| To install the ``sphinx`` Python package, you can use the following command
   
  .. code-block:: bash

     pip install --prefix=${VSC_DATA}/python_lib sphinx

* |VUB| |UG| |KUL| **Users with a different home institution**
        
  VSC_DATA is a file system at your local
  institution. Due to the distance between our clusters and your home institution, file access to 
  VSC_DATA will have a high latency. 
   
  Installing the packages that you need locally on **VSC_SCRATCH**
  will give much better performance.
```

Reply:
This extra space in the formatting was added deliberately to get the border on the left side of the blocks.
However, if it's more in line with the rest of the documentation, the suggestion is fine for me too.